general-configuration-guide/basic-setup.md

```diff
@@ -177,68 +138,83 @@ tinymce.init({
 
 ```js
 tinymce.init({
   selector: 'textarea',  // change this value according to your HTML
   menu: {
-    view: {title: 'Edit', items: 'cut, copy, paste'}
+    edit: {title: 'Edit', items: 'undo, redo, selectall'}
   }
 });
 ```
 
-Create your menu titles by adding a `title` value. The following example creates a menu titled "Happy". In the snippet below we create a menu with the title "Happy" and includes the "Source code" item.
+{% include codepen.html id="menu1" %}
+
+To create a menu titled "Happy", provide an identifier for the menu and an object with the `title` and `items` for the menu.
+
+For example:
 
 ```js
 tinymce.init({
   selector: 'textarea',  // change this value according to your HTML
   menu: {
-    view: {title: 'Happy', items: 'code'}
+    happy: {title: 'Happy', items: 'code'}
   },
-  plugins: 'code'  // required by the code menu item
+  plugins: 'code',  // required by the code menu item
+  menubar: 'happy'  // adds happy to the menu bar
 });
 ```
 
-Setting the value of `menubar` or `toolbar` to false removes them. The example below removes both the toolbar *and* menubar.
+{% include codepen.html id="menu2" %}
 
-```js
-tinymce.init({
-  selector: 'textarea',  // change this value according to your HTML
-  menubar: false,  // removes the menubar
-  toolbar: false  // removes the toolbar
-});
-```
+### Default menu controls
 
-> Important: Each page in the plugin documentation indicates via an icon if that plugin has toolbar buttons or menu items.
+The default menus are as follows:
 
+{% include configuration/defaultmenuitems.md %}
 
-## Basic configuration example
+> **Note**: Some menu items require a plugin, such as the `table` menu and items requires the `table` plugin.
 
-{{site.productname}} has many other configuration options available that enable further customization and extension of the editor to match the user's requirements.
+#### Additional information
 
-The full list of configuration options is in the [Configuration options references]({{ site.baseurl }}/configure/) section. Plugins, their options, and control associations are available in the [Plugins]({{ site.baseurl }}/plugins/) section.
+* For information on the `menu` setting, see: [User interface options - menu]({{site.baseurl}}/configure/editor-appearance/#menu).
+* For information on the `menubar` setting, see: [User interface options - menubar]({{site.baseurl}}/configure/editor-appearance/#menubar).
+* For a list of all available menu items, see: [Editor control identifiers - Menu controls]({{site.baseurl}}/advanced/editor-control-identifiers/#menucontrols).
 
-The following example is a walkthrough of a basic {{site.productname}} configuration.
+## Basic configuration example
+
+The following example is a basic {{site.productname}} configuration.
 
 ```html
 <!DOCTYPE html>
 <html>
 <head>
-  <script type="text/javascript" src='{{ site.cdnurl }}' referrerpolicy="origin"></script>
+  <script
```

Comment:
This is a weird way of writing a script, was it intential, or just auto formatter messing things up?

Reply:
Yeah, it's a bit weird. I did that to keep the line from flowing out of view on the website, eliminating the need to scroll sideways to view to the content of the codeblock